RUFAS/input_manager.py

```diff
@@ -111,14 +111,16 @@ def pool(self, incoming_pool: Dict[str, Any]) -> None:
         """The setter method for __pool"""
         self.__pool = incoming_pool
 
-    def set_metadata_depth_limit(self, limit: int) -> None:
-        """Override for the default metadata_depth_limit."""
+    def set_metadata_depth_limit(self, metadata_depth_limit: int) -> None:
```

Comment:
```suggestion
    def set_metadata_depth_limit(self, metadata_depth_limit: int | None) -> None:
```
When no GNU arg is passed, the metadata depth limit defaults to None - so it might be good to explicitly type that here.